src/main/java/com/dumbdogdiner/stickyapi/bukkit/command/tabcomplete/NonCompletingTabExecutor.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.command.tabcomplete;
+
+
+import com.dumbdogdiner.stickyapi.common.arguments.Arguments;
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collections;
+import java.util.List;
+
+public class NonCompletingTabExecutor implements TabExecutor {
```

Comment:
I feel like this just isn't necessary. We can return our own empty lists if needs be.

Reply:
It's a way of having one since it's used very often for quick access.